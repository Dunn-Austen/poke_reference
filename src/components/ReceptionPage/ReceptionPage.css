.reception-page {
  align-items: center;
  /* background-color: silver; */
  background: linear-gradient(rgba(0, 0, .5, .5), rgba(0, .1, .1, .1)), url('http://www.cartoondistrict.com/wp-content/uploads/2014/12/pokemon-wallpaper-HD-for-desktop-1.jpg');
  background-blend-mode: lighten saturation;
  background-repeat: no-repeat;
  background-size: 100% 100%;
  display: flex;
  flex-direction: column;
  height: 93vh;
  justify-content: center;
  width: 100%;
}

.reception-container {
  align-items: center;
  background-color: #C9C9C9;
  border: 8px ridge gold;
  border-radius: 20px;
  box-shadow: 0px 2px 25px 5px black;
  display: flex;
  flex-direction: column;
  height: 96%;
  justify-content: space-around;
}

.reception-text {
  background-color: #C9B03A;
  border: 5px solid #325F72;
  box-shadow: 0px 2px 4px 1px black;
  box-sizing: border-box;
  color: black;
  font-size: 18px;
  font-weight: lighten;
  text-align: center;
  /* text-shadow: 0px 3px 2px black; */
  width: 495px;
}

.home2 {
  background-color: gold;
  border: 2px solid #2A5060;
  box-shadow: 0px 2px 4px 1px black;
  margin-bottom: 5px;
}

.home2:hover {
  box-shadow: 0px 0px 4px 1px black;
}
