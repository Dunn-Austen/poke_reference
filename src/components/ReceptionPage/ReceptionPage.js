import React from 'react';
import { Link } from 'react-router-dom';
import './ReceptionPage.css';

const ReceptionPage = () => {
  return (
    <section className='reception-page'>
      <p className='reception-text'>
        Pokemon is a game based loosely off of natural history. Its many fictional creatures emulate those found
        in the real world. In playing pokemon, we encounter the fundamental joy of exploring the mysterious bounties of nature, of lifting up a rock to encounter
        the hidden world of what may lay beneath. To any outdoorsman, Pokemon offers this experience anew via fictional and fantastical ecosystems. You can once again, for the first time,
        explore a new world of earth-like possibilites. Except here, rodents emit lightning.

        It's important to understand that pokemon are type-based creatures powered by natural, elemental powers. Collecting these beings and pitting their skills against each other is the essence
        of the PokeWorld. Pokemon battles involve making the right stylistic matchups where having the appropriate
        elemental type to counter a specific foe confers a massive advantage. This app will help you leverage these natural balancing forces (water vs fire, fire vs grass) to optimize your Poke Battles.
        Search for any pokemon to find its types, the types best suited to exploit it, and a list of suggested pokemon wielding those particular skills.

        Step forward into the world of pokemon, where picking the RIGHT pokemon is key.
      </p>
      <Link to ='/pokesearch'>
        <button className='home'>Enter</button>
      </Link>
    </section>
  )
}

export default ReceptionPage;
