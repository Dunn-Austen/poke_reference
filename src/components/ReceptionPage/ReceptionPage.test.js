import React from 'react';
import { shallow } from 'enzyme';
import ReceptionPage from './ReceptionPage';

describe('ReceptionPage', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ReceptionPage />);
  });

  it('should match the ReceptionPage Snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
