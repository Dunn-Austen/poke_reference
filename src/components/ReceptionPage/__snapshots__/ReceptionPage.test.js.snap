// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReceptionPage should match the ReceptionPage Snapshot 1`] = `
<section
  className="reception-page"
>
  <div
    className="reception-container"
  >
    <p
      className="reception-text"
    >
      Pokemon is a game based loosely off of natural history. Its many fictional creatures emulate those found in the real world. In playing pokemon, we encounter the fundamental joy of exploring the mysterious bounties of nature, of lifting up a rock to encounter the hidden world of what may lay beneath. To any outdoorsman, Pokemon offers this experience anew via fictional and fantastical ecosystems. You can once again, for the first time, explore a new world of earth-like possibilites. Except here, rodents emit lightning.
    </p>
    <p
      className="reception-text"
    >
      It's important to understand that pokemon are type-based creatures powered by natural, elemental abilities. Collecting these beings and pitting their skills against each other is the essence of the PokeWorld. Pokemon battles involve making the right stylistic matchups where having the appropriate elemental type to counter a specific foe confers a massive advantage. This app will help you leverage these natural balancing forces (water vs fire, fire vs grass) to optimize your Poke Battles. Search for any pokemon to find its types, the types best suited to exploit it, and a list of suggested pokemon wielding those particular skills.
    </p>
    <p
      className="reception-text"
    >
      Step forward into the world of pokemon, where picking the RIGHT pokemon is key.
    </p>
    <Link
      to="/pokesearch"
    >
      <button
        className="home home2"
      >
        Enter
      </button>
    </Link>
  </div>
</section>
`;
